spec/factories/assessment_factory.rb

```diff
@@ -4,6 +4,11 @@
     remote_ip { Faker::Internet.ip_v4_address }
     submission_date { Date.current }
     matter_proceeding_type { 'domestic_abuse' }
+    version { '2' }
+
+    trait :with_latest_version do
+      version { '3' }
```

Comment:
A trait with the name `with_latest_version` is going to become out of date when we introduce version 4 (sad, but inevitible).  Why not just call it `:v3` ?

Reply:
Yeah I think that crossed my mind too and I forgot to change it 😂

Reply:
I think I was contemplating using latest version number from CFEConstants but let's keep it simple and do that 👍